### PR TITLE
Update IAM-premium-pack-readme.md

### PR DESCRIPTION
**Pull request by ShirleyDenkberg**
Added documentation for the app sync process.

## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added a description of the workflow and necessary configurations for the app syn process.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

**Review by idovandijk — COMMENTED** [review]
@ShirleyDenkberg I added a bit more to the `IAM - Sync User` playbook configuration, following a fix that we are releasing. Also, I reviewed your parts. Looks great. I made some small adjustments. Please see that they are written correctly. Thanks!

**Comment by ShirleyDenkberg** [comment]
@idovandijk I reviewed your changes and made some minor edits.

**Comment by idovandijk** [comment]
> @idovandijk I reviewed your changes and made some minor edits.

Looks great, we can merge the edits. I'm not sure why the build is failing. If this is not something that usually happens then need to reach out for help from content

File: content-repo/extra-docs/articles/IAM-premium-pack-readme.md [hunk 1]
```diff
@@ -1,13 +1,25 @@
+
 ---
```
Comment (yaakovi) [inline content-repo/extra-docs/articles/IAM-premium-pack-readme.md:2]:
```suggestion
---
```

**Review by idovandijk — APPROVED** [review]

**Pull request merged**